### PR TITLE
[API-150] Notify group members of new event creation

### PR DESCRIPTION
**Pull request by QMalcolm**
We may want to change this in the future because I can imagine many people will create a lot of events at once (especially considering we don't have repeating events yet). However, Let's not worry about it until it becomes a problem...

### CODE REVIEW

File: web/models/groupevents.ex [hunk 1]
```diff
@@ -39,5 +39,19 @@ defmodule Thegm.GroupEvents do
...
     |> cast(%{deleted: true}, [:deleted])
   end
+
+  def create_new_event_notification(event, groups_id, users_id) do
+    with {:ok, group} <- Thegm.Groups.get_group_by_id!(groups_id),
+      {:ok, recipients} <- Thegm.GroupMembers.get_group_member_ids(groups_id, [users_id]) do
+        event_date_string = event.start_time
+          |> DateTime.to_date
+          |> Date.to_erl
+          |> Timex.format!("{Mfull} {D}, {YYYY}")
```
Comment (SuperManifolds) [inline web/models/groupevents.ex:49]: This should use the user's date format configuration setting

**Review by SuperManifolds — CHANGES_REQUESTED** [review]
Email should use the user's date format configuration setting when possible

**Comment by QMalcolm** [comment]
This isn't an email, this is an internal notification, but your point still makes sense for this context. However there are a few problems, and I think it falls out of scope for the ticket.

1. There is no default for the date string preference. What am I to default to if a user does not have a preference saved?
2. I assume we are saving javascript datetime format strings? @trezy && @UncleClapton 
    a. Is there a predefined list of format strings available to the user or can they literally submit whatever they want?
3. The point above means that I'll have to build a translator from Javascript datetime format strings to Elixir Timex format strings.
    a. If it's a limited set, thats easy enough
    b. If it's literally anything then I have to build a universal translator for the conversions which seems like a rabbit hole.
    c. Either way this seems out of scope for this ticket.

In conclusion, what is being suggested seems out of scope for this ticket. I think that this PR should be approved and the ticket completed. Additionally, I think 4 tickets should be created.

1. Ticket in which we define defaults for every preference if possible
2. Ticket in which we limit the number of options for date format preferences
3. Ticket in which we build a translator for options defined in ticket 2
4. Ticket in which we use what happened tickets 1-3 to update this notification body to format the date to the users preference.

**Review by SuperManifolds — APPROVED** [review]

**Comment by SuperManifolds** [comment]
PR will be merged in its current state and behaviour will be corrected in a later ticket
